stdlib/JuliaSyntaxHighlighting.version

```diff
@@ -1,4 +1,4 @@
 JULIASYNTAXHIGHLIGHTING_BRANCH = main
-JULIASYNTAXHIGHLIGHTING_SHA1 = 19bd57b89c648592155156049addf67e0638eab1
+JULIASYNTAXHIGHLIGHTING_SHA1 = cdd5803653a0fee6952f4b48e62f07e9381d8676
```

Comment:
The current commit only exists in my PR for JuliaSyntaxHighlighting.jl.
(JuliaLang/JuliaSyntaxHighlighting.jl#9)

So `do not merge`.